scripts/build.js

```diff
@@ -130,15 +130,14 @@ function build(previousSizeMap) {
       console.log('You can control this with the ' + chalk.green('homepage') + ' field in your '  + chalk.cyan('package.json') + '.');
       console.log();
       console.log('The ' + chalk.cyan('build') + ' folder is ready to be deployed.');
-      console.log('To publish it at ' + chalk.green(homepagePath) + ', run:');
+      console.log('To publish it at ' + chalk.green(homepagePath) + ' using the docs/ folder, run:');
       console.log();
       console.log('  ' + chalk.cyan('git') + ' commit -am ' + chalk.yellow('"Save local changes"'));
-      console.log('  ' + chalk.cyan('git') + ' checkout -B gh-pages');
-      console.log('  ' + chalk.cyan('git') + ' add -f build');
+      console.log('  ' + chalk.cyan('rm') + ' -rf docs/');
```

Comment:
If `./docs/CNAME` file exists, we should preserve it. (Our current deployment code accidentally removes it, see #654.) When the custom domain is edited trough the GitHub Pages settings, GitHub saves it in this file.